### PR TITLE
Feature/ux

### PR DESCRIPTION
**Pull request by aaronmgdr**
Attempt to increase ux 
- add visual indication that map list are links
- add super basic navigation and conform to u- prefix for css utility classes
- add headings to park list

### CODE REVIEW

File: client/js/components/ParkList.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default class ParkList extends React.Component {
     render() {
         var parkList = this.props.parks.map((park) => {
             return (
-                <div className='row' onClick={() => this.selectPark(park)} key={park.park_id}>
+                <div className='row u-clickable' onClick={() => this.selectPark(park)} key={park.park_id}>
```
Comment (luqmaan) [inline client/js/components/ParkList.jsx]: Do you think it would be more semantic to make these `<button>`s or `<a>`s instead?

**Comment by luqmaan** [comment]
:deciduous_tree:

**Comment by luqmaan** [comment]
I'm going to merge this branch into my PR and take care of any conflicts there. Please merge this into master whenever you're ready.

**Comment by luqmaan** [comment]
Also, awesome stuff! Didn't even think about adding nav, but now we can add an about, terms, github link, etc.

File: client/js/components/ParkList.jsx (hunk 1 above)
Comment (aaronmgdr) [inline client/js/components/ParkList.jsx]: right after making the commit, i did think of it, actually.

**Comment by aaronmgdr** [comment]
about the Navigation. its pretty dumb right now. wasn't sure if we wanted to add some router or just change App component state through a prop function passed to Navigator

**Comment by luqmaan** [comment]
Yeah, we need a router badly. I was planning to just do something ghetto like this: https://github.com/luqmaan/instabus-react/blob/master/client/js/components/AppRouter.react.jsx

Hopefully theres a better (and equally simple) solution.

**Pull request merged**